### DISCUSSION TITLE
Migration of @ngneat/transloco to @jsverse/transloco

### DISCUSSION THREAD

**Discussion by andrii-3sdev** [ngneat/transloco#756] [General]
Hi everyone,

I'm trying to migrate my Angular project to the latest version of Transloco, but I noticed the namespace has changed from @ngneat to @jsverse. I couldn't find any documentation or discussion about this change.

Could anyone clarify the reason behind the namespace migration from @ngneat to @jsverse for transloco?

Are there any breaking changes or significant updates I should know before migrating to the new namespace? Are any migration guides or resources available to help me smoothly transition to the new namespace and the potentially new features or functionalities?

Thanks in advance for any information!

**Comment by shaharkazaz** [comment]
Hi @Andrii-TriasDev,

The move to `jsverse` was decided due to some reorganization that was made over at `ngneat`.
You can find all the breaking changes in the [breaking changes file](https://github.com/jsverse/transloco/blob/master/BREAKING_CHANGES.md), the organization rename was part of v7.

I'll find the time to write a blog post clarifying it.

I hope this answers your questions 🙂

**Comment by deedarb** [comment]
hi, what about ngneat/cashew package?

**Reply by shaharkazaz** [reply]
It's not related to Transloco in any way and will still be under ngneat.
